TemplateApplication/Account/AccountSheet.swift

```diff
@@ -26,7 +27,7 @@ struct AccountSheet: View {
                 if account.signedIn && !isInSetup {
                     AccountOverview(isEditing: $overviewIsEditing) {
                         NavigationLink {
-                            ContributionsList()
+                            ContributionsList(projectLicense: "MIT")
```

Comment:
Thank you; as noted in the meeting today it would be great to pass in the `LicenseType` and make it an enum-like struct. The general idea to use the Bundle name of the application as a default parameter is really cool, thank you for that!